alma/controllers/hook/PaymentOptionsHookController.php

```diff
@@ -149,8 +149,9 @@ public function run($params)
                         'desc' => $descPaymentButton,
                         'plans' => (array) $plans,
                         'deferred_trigger_limit_days' => $feePlans->$key->deferred_trigger_limit_days,
-                        'apiMode' => Settings::getActiveMode(),
+                        'apiMode' => strtoupper(Settings::getActiveMode()),
```

Comment:
why this change ?

Reply:
@Francois-Gomis 
It's because in old version we sent the mode api in lowercase and now we need to send in uppercase for In-page v2